### ISSUE TITLE
Feature: Move Output module to dlogg package

### ISSUE THREAD

**Issue by NCrashed** [Circular-Studios/Dash#149]
There was a conversation in gitter about long run need for logging subsystem ([dlogg](https://github.com/NCrashed/dlogg)), thus I'd like to start work on the subject.

Notes:
- logger should be stored as global variable as currently `OutputManager` is implemented
- keep `logDebug`, `logInfo`, and e.t.c. methods, but they would be a wrappers for static logger instance
- mapping from `Verbosity` and `OutputType` enums to corresponding dlogg enums

**Comment by dislogical** [comment]
If it would be easier, you could just outright replace most of the module, but leave it marked as deprecated. This way old code would still work, we can guide users toward the new system, and we don't have an alias spaghetti monster to deal with later.

**Comment by tmr9209** [comment]
Having an improved logging system would be much appreciated!  Currently there is no debugging, so I have been using our basic logging system a lot recently for animation.  Having a better one could potentially improve my progress.

**Comment by wozniakty** [comment]
Here's my input on the features I think would be ideal in a logging system, from a developer standpoint.  Feel free to implement it however you see fit:
- The developer tools would remain mostly the same kind of system: simple logging functions like logInfo, logDebug, etc.
- Along with that, messages I choose to log in code would still have varying levels, like they are now, with Info, Debug, Warning, etc.  These should of course have a clear hierarchy.
- In the Config.yml I'd want essentially 4 settings: Log Verbosity and Output Verbosity, one for both Debug and Release.
  - The way that would work, is I could essentially say Log everything in debug mode, output Error and Warning, and then Log only Error and Warning in release mode and Output only Error messages.
  - Thus, when I called logInfo (or w.e we labeled the method), it would use the appropriate verbosity for debug/release and then output and/or log the information accordingly.

Let me know if anything there is confusing/ambiguous
